Magento_Theme/styles/modules/_copyright.scss

```diff
@@ -0,0 +1,7 @@
+.copyright {
+    background-color: $gray-light;
+    color: $white;
+    text-align: center;
+    font-size: $font-size-extra-small;
+    padding: 10px 0;
+}
```

Comment:
can we change the name of this component for something more functional? to have a possibility to reuse it?